### PR TITLE
[Plot] Dates show 1970 when there is no data

### PR DESCRIPTION
**Pull request by akhenry**
[https://github.com/nasa/openmctweb/issues/80]()

### Description of changes

Added a check in the PlotTickGenerator to suppress labels if dimensions of plot are 0 (ie. no data is being shown). This prevents the Unix epoch from being erroneously displayed in this case.

### Author checklist

* Changes address original issue? Yes
* Unit tests included and/or updated with changes? Yes
* Command line build passes? Yes
* Expect to pass code review? Yes

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,605 @@
+0 info it worked if it ends with ok
```
Comment (larkin) [inline npm-debug.log]: You probably didn't want to check this in.  Probably don't even want history of it, so best to rewrite this commit.

Comment (akhenry) [inline npm-debug.log]: No, I did not :) I'm going to add it to gitignore for future cases.

File: platform/features/plot/src/SubPlot.js [hunk 2]
```diff
@@ -64,6 +64,16 @@ define(
             this.updateTicks();
         }
 
+        /**
+         * Tests whether this subplot has domain data to show for the current pan/zoom level. Absence of domain data
+         * implies that there is no range data displayed either
+         * @returns {boolean} true if domain data exists for the current pan/zoom level
+         */
```
Comment (VWoeltjen) [inline platform/features/plot/src/SubPlot.js]: I'd mark this as `@private` since it's not used from anywhere outside of SubPlot

**Comment by VWoeltjen** [comment]
### Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? Y
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y

**Pull request merged**